src/app/banners/BannerWrapper.tsx

```diff
@@ -10,7 +10,7 @@ import { useEffect, useMemo, useState } from 'react';
 import SubscriptionBanner from './SubscriptionBanner';
 import { userSelectors } from 'app/store/slices/user';
 
-const OFFER_END_DAY = new Date('2025-04-08');
+const OFFER_END_DAY = new Date('2025-04-30');
```

Comment:
In the title of the PR it says campaign from the 8th to the 18th, is this date correct?